R/pkg/R/DataFrame.R

```diff
@@ -2880,12 +2888,15 @@ setMethod("unionAll",
 #' df1 <- select(createDataFrame(mtcars), "carb", "am", "gear")
 #' df2 <- select(createDataFrame(mtcars), "am", "gear", "carb")
 #' head(unionByName(df1, df2))
+#'
+#' df3 <- select(createDataFrame(mtcars), "carb")
+#' head(unionByName(df1, df3))
```

Comment:
an example where `allowMissingColumns=TRUE` would be helpfl

Reply:
Thanks, this is suppose to be the one. Fixed.